emigration.lua

```diff
@@ -1,22 +1,29 @@
 --Allow stressed dwarves to emigrate from the fortress
 -- For 34.11 by IndigoFenix; update and cleanup by PeridexisErrant
 -- old version:  http://dffd.bay12games.com/file.php?id=8404
---@module = true
---@enable = true
+--[====[
```

Comment:
docs have moved to separate .rst files. If you made any changes to the docs, they should go in https://github.com/DFHack/scripts/blob/master/docs/emigration.rst
script files shouldn't have duplicate docs since they just get out of date.

Reply:
I don't think anything relevant to the docs has changed.

Though I don't know if the part in bold is technically true: "Dwarves who can leave in friendly company (e.g. a dwarven merchant caravan) will choose to do so, but extremely stressed dwarves can choose to leave alone, **or even in the company of a visiting elven bard!**", but I suppose it hasn't been true in many years.

Perhaps it's not intended literally, but as a description of how despaired some of the emigrating dwarves must be?